cloud/pkg/apis/devices/v1alpha2/device_instance_types.go

```diff
@@ -287,20 +287,17 @@ type BluetoothReadConverter struct {
 // Specify the operation that should be performed to convert incoming data into understandable form
 type BluetoothOperations struct {
 	// Required: Specifies the operation to be performed to convert incoming data
-	BluetoothOperationType BluetoothArithmeticOperationType `json:"operationType,omitempty"`
+	BluetoothOperationType string `json:"operationType,omitempty"`
 	// Required: Specifies with what value the operation is to be performed
 	BluetoothOperationValue float64 `json:"operationValue,omitempty"`
 }
 
-// Operations supported by Bluetooth protocol to convert the value being read from the device into an understandable form
-type BluetoothArithmeticOperationType string
```

Comment:
Let's keep the type definition here according to the code specification :) And when user set value to the BluetoothOperationType, only the values in the const below are allowed.

Reply:
ok